packages/wp-dashboard/src/components/editorSettings/adManagement/googleAdManager/stories/index.js

```diff
@@ -29,20 +25,11 @@ export default {
   args: {
     slotId: '',
   },
-  parameters: {
-    controls: {
-      include: ['slotId'],
-    },
+  argTypes: {
+    handleUpdate: { action: 'update google ad manager' },
   },
 };
 
 export const _default = (args) => {
-  return (
-    <GoogleAdManager
-      handleUpdate={(newSlotId) =>
-        action('update google ad manager')(newSlotId)
-      }
-      {...args}
-    />
-  );
+  return <GoogleAdManager {...args} />;
```

Comment:
Same here: I'll add this to the same ticket
![image](https://user-images.githubusercontent.com/22185279/152850268-56c20945-2dd6-4097-8397-47dfc4f55c24.png)